types/time.go

```diff
@@ -21,8 +21,7 @@ func (t *Time) UnmarshalJSON(data []byte) error {
 		s = s[1 : len(s)-1]
 	}
 
-	switch s {
-	case "null", "0", "":
+	if s == "null" || strings.Trim(s, "0.") == "" {
```

Comment:
```suggestion
	if s == "" || s[0] == 'n' || s[0] == '0' {
```

more performant and if it starts with 0 it just thinks its cooked. 😆